analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/RawTextParser.kt

```diff
@@ -106,31 +100,32 @@ class RawTextParser(
 
         if (!withoutDefaultExcludes) exclude += DEFAULT_EXCLUDES
 
-        val results: Map<String, FileMetrics> =
-            MetricCollector(inputFile!!, exclude, fileExtensions, metricNames, verbose, maxIndentLvl, tabWidth).parse()
+        val projectMetrics: ProjectMetrics =
+            ProjectMetricsCollector(inputFile!!, exclude, fileExtensions, metricNames, verbose, maxIndentLvl, tabWidth).parseProject()
         println()
 
-        if (results.isEmpty()) {
+        if (projectMetrics.metricsMap.isEmpty()) {
```

Comment:
We could introduce a method in ProjectMetrics for this check.
```suggestion
        if (projectMetrics.isEmpty()) {
```